README.md

```diff
@@ -1 +1,31 @@
-# Lab19-MessageQueues
\ No newline at end of file
+# Lab18-Message Queues
+
+## Message Queues
+
+### Author: Shubham Majumdar
+
+### Links and Resources
+* [submission PR](https://github.com/Shubham-401n16/Lab19-MessageQueues/pull/1)
+* [Github Actions]
+
+#### Documentation
+* [api docs](http://xyz.com/api-docs) (API servers)
+* JSDoc Comment Required for all modules and functions in all .js files
+
+### Setup
+#### `.env` requirements (where applicable)
+* `PORT` -
+* `MONGODB_URI` -
+
+#### How to initialize/run your server app (where applicable)
+* `npm start`
+* npm init
+* npm i express socket.io socket.io-client cors morgan
+* node api.js
+  
```

Comment:
Should also include how to run the other files, and the route to use for POST. For example "In Postman do a POST and  `delivery/:vendor/:orderId` for example for the candy shop `/delivery/candy-shop/12345-ABCD` "